spec/index.bs

```diff
@@ -2169,6 +2173,34 @@ interface GPUTextureUsage {
 };
 </script>
 
+<div algorithm>
+    <dfn abstract-op>maximum mipLevel count</dfn>(dimension, size)
+    **Arguments:**
+        - {{GPUTextureDescriptor/dimension}} |dimension|
+        - {{GPUTextureDescriptor/size}} |size|
+
+    1. Calculate the values of |w|, |h|, and |d|:
+        - If |dimension| is:
+            <dl class="switch">
+                : {{GPUTextureDimension/"1d"}}
+                :: Let |w| = |size|.[=Extent3D/width=].
+                :: Let |h| = 1.
+                :: Let |d| = 1.
+
+                : {{GPUTextureDimension/"2d"}}
+                :: Let |w| = |size|.[=Extent3D/width=].
+                :: Let |h| = |size|.[=Extent3D/height=].
+                :: Let |d| = 1.
+
+                : {{GPUTextureDimension/"3d"}}
+                :: Let |w| = |size|.[=Extent3D/width=].
+                :: Let |h| = |size|.[=Extent3D/height=].
+                :: Let |d| = |size|.[=Extent3D/depthOrArrayLayer=].
+            </dl>
+    1. Let |m| = the maximum value of |w|, |h|, and |d|.
```

Comment:
nit: the switch could simply define `m`, instead of `w`, `h`, and `d`. e.g.

: "2d"
:: Let |m| = max(size.width, size.height)

Reply:
Done.